src/app/parcels/ParcelsPage.tsx

```diff
@@ -286,11 +292,10 @@ const ParcelsPage: React.FC<{}> = () => {
             primaryFilters: Filter<ParcelsTableRow, any>[];
             additionalFilters: Filter<ParcelsTableRow, any>[];
         }> => {
-            const startOfToday = dayjs().startOf("day");
-            const endOfToday = dayjs().endOf("day");
+            const today = dayjs().utc();
```

Comment:
I don't think we want to use utc here - we'd probably want to use "today's date" for the user which would be BST or UTC depending on season. `dayjs()` should just provide the current time in the current timezone
```suggestion
            const today = dayjs();
```

Reply:
removed